.buildkite/x-pack/pipeline.xpack.dockerlogbeat.yml

```diff
@@ -22,34 +27,67 @@ env:
   PACKAGING_ARM_PLATFORMS: "linux/arm64"
   PACKAGING_PLATFORMS: "+all linux/amd64 linux/arm64 windows/amd64 darwin/amd64 darwin/arm64"
 
-  #Deps
-  ASDF_MAGE_VERSION: 1.15.0
-
 steps:
 
-  - input: "Input Parameters"
-    key: "force-run-stages"
-    fields:
-    - select: "Dockerlogbeat - run_xpack_dockerlogbeat"
-      key: "run_xpack_dockerlogbeat"
-      options:
-        - label: "True"
-          value: "true"
-        - label: "False"
-          value: "false"
-      default: "false"
-
-    if: "build.source == 'ui'"
-
-  - wait: ~
-    if: "build.source == 'ui'"
-    allow_dependency_failure: false
-
-  - label: ":linux: Load dynamic x-pack dockerlogbeat pipeline"
-    key: "xpack-dockerlogbeat-pipeline"
-    command: ".buildkite/scripts/generate_xpack_dockerlogbeat_pipeline.sh"
-    agents:
-      image: "docker.elastic.co/ci-agent-images/platform-ingest/buildkite-agent-beats-ci:latest"
-    notify:
-      - github_commit_status:
-          context: "${BEATS_PROJECT_NAME}: Load dynamic pipeline's steps"
+  - group: "Xpack/Dockerlogbeat Mandatory Tests"
+    key: "xpack-dockerlogbeat-mandatory-tests"
+    steps:
+      - label: ":ubuntu: Xpack/Dockerlogbeat Ubuntu Unit Tests"
+        key: "mandatory-linux-unit-test"
+        command: "cd $BEATS_PROJECT_NAME && mage build unitTest"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_UBUNTU_X86_64}"
+          machineType: "${GCP_DEFAULT_MACHINE_TYPE}"
+        artifact_paths:
+          - "$BEATS_PROJECT_NAME/build/*.xml"
+          - "$BEATS_PROJECT_NAME/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "$BEATS_PROJECT_NAME: Ubuntu Unit Tests"
+
+      - label: ":ubuntu: Xpack/Dockerlogbeat Go Integration Tests"
+        key: "mandatory-int-test"
+        command: "cd $BEATS_PROJECT_NAME && mage goIntegTest"
+        env:
+          MODULE: $MODULE
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_UBUNTU_X86_64}"
+          machineType: "${GCP_DEFAULT_MACHINE_TYPE}"
+        artifact_paths:
+          - "$BEATS_PROJECT_NAME/build/*.xml"
+          - "$BEATS_PROJECT_NAME/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "$BEATS_PROJECT_NAME: Go Integration Tests"
+
+  - group: "Xpack/Dockerlogbeat Packaging"
+    key: "xpack-dockerlogbeat-packaging"
+    if: build.env("BUILDKITE_PULL_REQUEST") != "false"
```

Comment:
see the conditions how its been done in https://github.com/elastic/beats/pull/38827/files#diff-87cc9e7aecd3c1a6382e951a1b17429443aa3f12387328521fa06b2c8f83a814R234 for executing the packaging step.